### DISCUSSION TITLE
Suggestion: Storage space requirements

### DISCUSSION THREAD

**Discussion by TeddyBearJosh** [carson-katri/dream-textures#309] [Ideas]
Without enough space on the C drive, the installation will fail and have missing scripts.

When installing, it would be nice for the software to have a storage check, in which is allocates space for the .zip and weights (combined total of 16gb)

**Comment by carson-katri** [comment]
I think this would need to be implemented in Blender, not the addon.

**Comment by TeddyBearJosh** [comment]
Even better idea, thankyou 😊 hopefully Blender can implement something
like this 👍
